### ISSUE TITLE
Crash fighting slime with warrior (parallel universe)

### ISSUE THREAD

**Issue by jepler** [TerryCavanagh/diceydungeons.com#1070]
```AudioControl.hx:115: Event _poison does not exist
AudioControl.hx:115: Event _poison does not exist
CrashDumper.hx:248: CRASH session.id = dicey_dungeons_2019-05-31_17'02'36
CrashDumper.hx:249: MESSAGE = --------------------------------------
filename:	diceydungeons
package:	com.terrycavanaghgames.diceydungeons
version:	0.17.2 / 0.7.0 (mod API)
sess. ID:	dicey_dungeons_2019-05-31_17'02'36
started:	2019-05-31 17:02:36
--------------------------------------
crashed:	2019-05-31 17:04:44
duration:	00:02:07
error:		ERROR in callscenemethod(Combat,update) static : Null Object Reference, stack = 
Called from elements.Fighter.applyequipmentcurses (elements/Fighter.hx line 735)
Called from elements.CombatCommand.execute (elements/CombatCommand.hx line 219)
Called from states.Combat.update (states/Combat.hx line 207)
Called from Reflect.callMethod (/usr/share/haxe/std/cpp/_std/Reflect.hx line 55)
stack:
haxegon.Scene.callscenemethod (haxegon/Scene.hx line 93)
haxegon.Scene.update (haxegon/Scene.hx line 53)
haxegon.Core.doupdate (haxegon/Core.hx line 268)
haxegon.Core.onEnterFrame (haxegon/Core.hx line 182)
starling.events.EventDispatcher.__invokeEvent (starling/events/EventDispatcher.hx line 184)
starling.events.EventDispatcher.dispatchEvent (starling/events/EventDispatcher.hx line 144)
starling.display.DisplayObject.dispatchEvent (starling/display/DisplayObject.hx line 778)
starling.display.DisplayObjectContainer.broadcastEvent (starling/display/DisplayObjectContainer.hx line 449)
starling.display.Stage.advanceTime (starling/display/Stage.hx line 108)
starling.core.Starling.advanceTime (starling/core/Starling.hx line 469)
starling.core.Starling.nextFrame (starling/core/Starling.hx line 455)
starling.core.Starling.onEnterFrame (starling/core/Starling.hx line 682)
openfl.events.EventDispatcher.__dispatchEvent (openfl/events/EventDispatcher.hx line 443)
openfl.display.DisplayObject.__dispatch (openfl/display/DisplayObject.hx line 1236)


CrashDumper.hx:142: onError(EOF)
```

**Comment by ThatShad0wGuy** [comment]
I also encountered this same issue in the most recent version (v1.4.3). Error log:

--------------------------------------
filename:	diceydungeons
package:	com.terrycavanaghgames.diceydungeons
version:	v1.4.3 / 0.11.0 (mod API)
sess. ID:	dicey_dungeons_2019-09-25_23'00'10
started:	2019-09-25 23:00:10
--------------------------------------
crashed:	2019-09-25 23:06:52
duration:	00:06:42
error:		ERROR in callscenemethod(Combat,update) static : Null Object Reference, stack = 
Called from elements.Fighter.applyequipmentcurses (elements/Fighter.hx line 868)
Called from elements.CombatCommand.execute (elements/CombatCommand.hx line 348)
Called from states.Combat.update (states/Combat.hx line 270)
Called from Reflect.callMethod (C:\coding\haxe\haxe\std/cpp/_std/Reflect.hx line 55)
stack:
haxegon.Scene.callscenemethod (haxegon/Scene.hx line 93)
haxegon.Scene.update (haxegon/Scene.hx line 53)
haxegon.Core.doupdate (haxegon/Core.hx line 268)
haxegon.Core.onEnterFrame (haxegon/Core.hx line 182)
starling.events.EventDispatcher.__invokeEvent (starling/events/EventDispatcher.hx line 184)
starling.events.EventDispatcher.dispatchEvent (starling/events/EventDispatcher.hx line 144)
starling.display.DisplayObject.dispatchEvent (starling/display/DisplayObject.hx line 778)
starling.display.DisplayObjectContainer.broadcastEvent (starling/display/DisplayObjectContainer.hx line 449)
starling.display.Stage.advanceTime (starling/display/Stage.hx line 108)
starling.core.Starling.advanceTime (starling/core/Starling.hx line 469)
starling.core.Starling.nextFrame (starling/core/Starling.hx line 455)
starling.core.Starling.onEnterFrame (starling/core/Starling.hx line 682)
openfl.events.EventDispatcher.__dispatchEvent (openfl/events/EventDispatcher.hx line 443)
openfl.display.DisplayObject.__dispatch (openfl/display/DisplayObject.hx line 1236)

If I had to guess, I suspect there's an issue with how Fury? (specifically, how it disables equipment) interacts with Poison?.
